librabbitmq/amqp_socket.c

```diff
@@ -57,7 +57,13 @@
 
 #include <errno.h>
 
-#ifdef _WIN32
+#if (defined (__MINGW32__) || defined (__MINGW64__))
```

Comment:
Same here, don't duplicate the whole block, change the preprocessor conditional and correct the capitalization of the include files.